### DISCUSSION TITLE
Occurrence active response

### DISCUSSION THREAD

**Discussion by CISO-GYM** [wazuh/wazuh#21432] [Q&A]
I want to block someone when trying to login to a user in invalid credentials or invalid username (rule 5710-5760) but after 3 attempts not from the first one, can i change the occurrence?

**Comment by 23uk** [comment]
You should use  frequency parameter. 
Try this:

`<rule id="5760" level="5" frequency="3" overwrite="yes">`
`<if_sid>5700,5716</if_sid>`
`<match>Failed password|Failed keyboard|authentication error</match>`
`<description>sshd: authentication failed.</description>`
`</rule>`

**Reply by CISO-GYM** [reply]
What is the overwrite used for?

**Comment by 23uk** [comment]
Without overwrite, rule from /var/ossec/rules will be triggered.

**Reply by CISO-GYM** [reply]
Sorry for so many questions, but where do i write this script in? 
<rule id="5760" level="5" frequency="3" overwrite="yes">
<if_sid>5700,5716</if_sid>
<match>Failed password|Failed keyboard|authentication error</match>
<description>sshd: authentication failed.</description>
</rule>
because in the command and active response section they do not have the frequency parameters

**Comment by 23uk** [comment]
create file with any name /var/ossec/etc/rules/*.xml
